gauges/replication.go

```diff
@@ -51,30 +51,33 @@ func (g *Gauges) StreamingWALs() prometheus.Gauge {
 	)
 }
 
-// ReplicationLag returns a prometheus gauge for the database replication
-// lag in milliseconds
-func (g *Gauges) ReplicationLag() prometheus.Gauge {
+// ReplicationDelay returns a prometheus gauge for the database replication
+// lag in seconds
+func (g *Gauges) ReplicationDelay() prometheus.Gauge {
+	return g.new(
+		prometheus.GaugeOpts{
+			Name:        "postgresql_replication_delay",
```

Comment:
It's better to name this metric as `postgresql_replication_delay_seconds` to follow prometheus naming best practices